codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHTTPBindingProtocolGenerator.kt

```diff
@@ -110,31 +106,14 @@ abstract class AWSHttpBindingProtocolGenerator : HttpBindingProtocolGenerator()
         defaultTimestampFormat: TimestampFormatTrait.Format,
         path: String
     ) {
-        val decoder = StructDecodeGenerator(ctx, members, writer, defaultTimestampFormat, path)
-        decoder.render()
+        StructDecodeGenerator(ctx, shapeContainingMembers, members, shapeMetaData, writer).render()
     }
 
     override fun addProtocolSpecificMiddleware(ctx: ProtocolGenerator.GenerationContext, operation: OperationShape) {
         operationMiddleware.appendMiddleware(operation, OperationEndpointResolverMiddleware(ctx))
         operationMiddleware.appendMiddleware(operation, UserAgentMiddleware(ctx.settings))
     }
 
-    override fun generateMessageMarshallable(ctx: ProtocolGenerator.GenerationContext) {
-        var streamingShapes = outputStreamingShapes(ctx)
-        val messageUnmarshallableGenerator = MessageUnmarshallableGenerator(ctx)
-        streamingShapes.forEach { streamingMember ->
-            messageUnmarshallableGenerator.renderNotImplemented(streamingMember)
-        }
-    }
-
-    override fun generateMessageUnmarshallable(ctx: ProtocolGenerator.GenerationContext) {
-        val streamingShapes = inputStreamingShapes(ctx)
-        val messageMarshallableGenerator = MessageMarshallableGenerator(ctx, defaultContentType)
-        for (streamingShape in streamingShapes) {
-            messageMarshallableGenerator.renderNotImplemented(streamingShape)
-        }
-    }
-
```

Comment:
The default `generateMessageMarshallable()` and `generateMessageUnmarshallable()` implementations are usable here.

Reply:
This is not true; this class inherits from `HTTPBindingProtocolGenerator` which declares these 2 methods as abstract.  So, implementations are required here.